features/user_management.feature

```diff
@@ -0,0 +1,12 @@
+# Feature: As a visitor
+#   So that I can log in to the system and place orders
+#   I would like to see a 'log in' or 'register' button on the home page.
+#
+# Scenario: Allows a visitor to access a registration page
+#   Given I am on the "home page"
+#   And I click on the "Log In" link
+#   Then I should be on the 'login' page
+#   When I fill in "Username:" with "user"
+#   And I fill in "Password:" with "user"
+#   And I click "Log In" button
+#   Then I should see "Successfully logged in user"
```

Comment:
So I know my colleagues have something like this in all of their computers. I didn't want to just delete my copy and not acknowledge that there's a problem. I know this isn't something one would ordinarily want to see, but we all have this test running and none of us should after we merge user-registration. I don't know the best way to resolve this, really.